ethereumj-core/src/main/java/org/ethereum/net/server/Channel.java

```diff
@@ -372,8 +375,28 @@ public void prohibitTransactionProcessing() {
         eth.disableTransactions();
     }
 
-    public void sendTransaction(List<Transaction> tx) {
-        eth.sendTransaction(tx);
+    /**
+     * Send transactions from input to peer corresponded with channel
+     * Using {@link #sendTransactionsSafely(List)} is recommended instead
+     * @param txs   Transactions
+     */
+    public void sendTransactions(List<Transaction> txs) {
+        eth.sendTransaction(txs);
+    }
+
+    /**
+     * Sames as {@link #sendTransactions(List)} but input list is randomly sliced to
+     * contain not more than {@link MAX_SAFE_TXS} if needed
+     * @param txs   List of txs to send
+     */
+    public void sendTransactionsSafely(List<Transaction> txs) {
```

Comment:
`Safely` sounds too abstract for me and confusing on the other hand (is regular `sendTransactions` unsafe??)